### ISSUE TITLE
Custom command params substitution

### ISSUE THREAD

**Issue by joihn** [jesseduffield/lazydocker#449]
thanks for this tool. small issue:

**Describe the bug**
on latest master (as of 12may), custom command parameter subsitution is not working. The bug does **not** appear on latest relase.

**To Reproduce**
1. build from master as of 12 may 
2. use this example custom command
```
customCommands:
  containers:
    - name: bash
      attach: true
      command: 'docker exec -it {{ .Container.ID }} bash'
      serviceNames: []
```
3. try to launch the custom command
`Container.ID` does not get correctly subsituted. it stay as it when sent to bash

ENV:
ubuntu 20.04, x86

**Comment by mark2185** [comment]
I think it's the same [what I stumbled upon](https://github.com/jesseduffield/lazydocker/pull/387#issuecomment-1443645464), seems to be introduced by #387 .

**Comment by ngocphamm** [comment]
And now I think it get into the latest release `0.21.0`.

I have this custom command trying to update a service's image by doing `docker compose pull && docker compose up`.

```
customCommands:
  services:
    - name: update-service
      attach: true
      command: '{{ .DockerCompose }} pull {{ .Service.Name }} && {{ .DockerCompose }} up {{ .Service.Name }}'
```

Run it and I get this

![CleanShot 2023-07-21 at 14 46 20@2x](https://github.com/jesseduffield/lazydocker/assets/395123/785de3b7-de61-47a9-9fe3-494c95c5f8f8)

**Comment by VaultVulp** [comment]
For anyone wondering how to downgrade back to `0.20.0` via `homebrew` - you can use the following snippet:
```bash
wget https://raw.githubusercontent.com/jesseduffield/homebrew-lazydocker/b6d29bcd59058d1b7ec371df12f75d7048b40789/lazydocker.rb
brew uninstall lazydocker
brew install -s lazydocker.rb
rm ./lazydocker.rb
```

**Comment by jesseduffield** [comment]
This should now be fixed in https://github.com/jesseduffield/lazydocker/releases/tag/v0.21.1 (waiting on homebrew to merge it). Sorry it took so long! Thanks to @mark2185  for the fix

**Comment by mark2185** [comment]
I believe it has been [merged into homebrew](https://formulae.brew.sh/formula/lazydocker) so this issue can be closed.
